### ISSUE TITLE
Neuronales Netzwerk verbessern

### ISSUE THREAD

**Issue by AgenoDrei** [AgenoDrei/DetectingDiabeticRetinopathyFromSmartphoneVideoFundus#8]
Ideen:
- [x] Pretraining mit kaggle Daten
- [x] Verwendung Resnet / Inception Net
- [x] Learning Rate decay
- [ ] Hervorhebung der Netzwerk Aktivierung
- [x] Confidence abschätzen.
- [ ] Regression für die Vorhersage ausprobieren.
- [x] Weight decay?
